.github/workflows/ping-other-repos.yml

```diff
@@ -0,0 +1,25 @@
+name: Remote Dispatch Action Initiator
+
+on:
+  push:
+  repository_dispatch:
+
+jobs:
+  ping-pong:
+    runs-on: ubuntu-latest
+    steps:
+      - name: PING - Dispatch initiating repository event
+        if: github.event.action != 'pong'
+        run: |
```

Comment:
I'm not exactly sure if we should—or if you've already considered this—but maybe we ought to use an off-the-shelf step for this? Something like https://github.com/marketplace/actions/repository-dispatch would be a little more readable, at least.

Reply:
> I'm not exactly sure if we should—or if you've already considered this—but maybe we ought to use an off-the-shelf step for this? Something like [github.com/marketplace/actions/repository-dispatch](https://github.com/marketplace/actions/repository-dispatch) would be a little more readable, at least.

I’m looking into that right now

Reply:
> > I'm not exactly sure if we should—or if you've already considered this—but maybe we ought to use an off-the-shelf step for this? Something like [github.com/marketplace/actions/repository-dispatch](https://github.com/marketplace/actions/repository-dispatch) would be a little more readable, at least.
> 
> I’m looking into that right now

OK — thanks much for suggesting; it’s definitely way more readable, so I’ve switched the source to using that intead